tools/pip/setup.py

```diff
@@ -97,6 +97,8 @@ def has_ext_modules(self):
                 os.path.join(CURRENT_DIR, 'mxnet/include/mshadow'))
 shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/tvm/nnvm/include/nnvm'),
                 os.path.join(CURRENT_DIR, 'mxnet/include/nnvm'))
+shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/include'),
+                os.path.join(CURRENT_DIR, 'mxnet/include/mkldnn'))
```

Comment:
this should only be needed for the pip versions with mkl

Reply:
That's correct. Will change it.